doc/ref_model/chapters/chapter01.rst

```diff
@@ -0,0 +1,99 @@
+Introduction
+============
+
+Overview
+--------
+
+The Reference Model (RM) specifies a virtualisation technology agnostic (VM-based and container-based) cloud infrastructure abstraction and acts as a "catalogue" of the exposed infrastructure capabilities, resources, and interfaces required by the workloads. This document has been developed by the Linux Foundation Networking project `Anuket <../../common/chapter00.md>`__.
+
+**Problem Statement:** Based on community consultations, including telco operators, technology suppliers, and software developers, there is a realisation that there are significant technical, operational and business challenges to the development and deployment of Virtual Network Function (VNF)/ Cloud Native Network Function (CNF) network applications related to the lack of a common cloud infrastructure platform. These include but are not limited to the following:
```

Comment:
I realise that this is rst conversion but this definition needs changes. Suggest the following -- may be made in another PR: the issues is the use of "network applications" after VNF and CNF:
```suggestion
**Problem Statement:** Based on community consultations, including telco operators, technology suppliers, and software developers, there is a realisation that there are significant technical, operational, and business challenges to the development and deployment of Virtual Network Functions (VNF) and Cloud Native Network Functions (CNF) due to the lack of a common cloud infrastructure platform. These include but are not limited to the following:
```